src/convert/convertContext.ts

```diff
@@ -273,7 +273,7 @@ class NonDecompositionFinalizer extends ConvertTransactionFinalizer<NonDecomposi
    */
   private getAllComponentsOfType(defaultDirectory: string, componentType: string): SourceComponent[] {
     // assumes that defaultDir is one level below project dir
```

Comment:
yes, the difference now is that it doesn't strip the last member in `defaultDirectory`, so for `force:mdapi:convert -r mdapi -d test1/test2`

```ts
const defaultDirectory = '/Users/cdominguez/code/sfdx-project/test1/test2';
const projectDir = resolve(defaultDirectory);

// projectDir would equal '/Users/cdominguez/code/sfdx-project/test1/test2'
```

...:thinking: wouldn't this cause SDR fail to find customlabels if you have multiple MPDs? Since `projectDir` will not be pointing to the root project dir.

this is the only place it's being called: https://github.com/forcedotcom/source-deploy-retrieve/blob/7804d447bb2905304b011f2f2ac16ce6037575f1/src/convert/convertContext.ts#L198